src/components/Assets/AssetsFilter.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FilterComponent from '../common/FilterComponent';
+import FilterButton from '../common/FilterButton';
+
+const AssetsFilterButton = props => (
```

Comment:
Instead of creating another filter button, move [this one](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/components/User/UserFilter.jsx) to `components/common` directory and use it. If you look at the existing filter and the one you have just created, they have the same props so it can be reused.

Don't forget to update the import in the user header file

Reply:
Noted.